src/RulesTest/MonsterTest.cs

```diff
@@ -1021,15 +1020,5 @@ public void MonsterUserControl()
             var results = Axe.Windows.Rules.Rules.RunAll(e);
             return results.ToDictionary(r => r.RuleInfo.ID, r => r.EvaluationCode);
         }
-
-        private static void WriteResultsToDebugOutput(Dictionary<RuleId, EvaluationCode> results)
```

Comment:
Please don't remove this, I sometimes need it when rules change and the monster tests fail.

Reply:
Restored